### ISSUE TITLE
JExcel Array push

### ISSUE THREAD

**Issue by indunil-amarasinghe** [jspreadsheet/ce#1028]
Hi Paul,
Is there a way to push arrays using jexcel in javascript. Please be kind enough to let me know.

The push method should work exactly the way it works with arrays.

Thanks.

**Comment by pphod** [comment]
Can you rephrase? Sorry did understand what you are asking.

**Comment by indunil-amarasinghe** [comment]
Hi Paul,
I need a reply to the following question.

https://stackoverflow.com/questions/62558374/pushing-arrays-using-jexcel-plugin-to-customize-a-subtitle-editor
